server/src/pages/shotindex/view.js

```diff
@@ -107,6 +107,13 @@ class Body extends React.Component {
     const nextPageNumber = this.props.pageNumber - 0 + 1;
     const nextClasses = ["shots-page-nav"].concat(!hasNext && "disabled").join(" ");
     const hidden = totalPages < 2;
+    let arrowheadPrev = this.props.staticLink("/static/img/arrowhead-left-16.svg");
+    let arrowheadNext = this.props.staticLink("/static/img/arrowhead-right-16.svg");
+    if (document.dir === "rtl") {
+      const save = arrowheadNext;
+      arrowheadNext = arrowheadPrev;
+      arrowheadPrev = save;
```

Comment:
Doesn't really matter, but you can use fancy JS here, like `[arrowheadNext, arrowheadPrev] = [arrowheadPrev, arrowheadNext]`